### PR TITLE
Notes field of Run cleanup

### LINKED ISSUE #250: PublicRun fields remake
### Description

There are now more attributes added to Run in the database, that need to be reflected in the PublicRun type. There are also unsupported fields, that should be removed. 

### Acceptance Criteria

Run is tested using Postman, and client run updates correctly. 

### Proposed Solution

Change the PublicRun type in scylla, along with transformer. Change type in client.

### PR DESCRIPTION
**Pull request by bracyw**
## Changes

Added a note field to PublicRun (changed transformer, PublicRun type (CL and scylla)).

Made note field able to be NULL. 

## Notes

_Any other notes go here_

## Test Cases

new run button still works 

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] All commits are tagged with the ticket number
- [ ] No linting errors / newline at end of file warnings
- [ ] All code follows repository-configured prettier formatting
- [ ] No merge conflicts
- [ ] All checks passing
- [ ] Screenshots of UI changes (see Screenshots section)
- [ ] Remove any non-applicable sections of this template
- [ ] Assign the PR to yourself
- [ ] No `package-lock.json` changes (unless dependencies have changed)
- [ ] Request reviewers & ping on Slack
- [ ] PR is linked to the ticket (fill in the closes line below)

Closes #250

### CODE REVIEW

File: scylla-server/src/schema.rs [hunk 1]
```diff
@@ -25,7 +25,7 @@ diesel::table! {
         latitude -> Nullable<Float8>,
         longitude -> Nullable<Float8>,
         driverName -> Nullable<Text>,
-        notes -> Text,
+        notes -> Nullable<Text>,
```
Comment (RChandler234) [inline scylla-server/src/schema.rs]: notes shouldn't be optional :(

**Review by RChandler234 — CHANGES_REQUESTED** [review]

**Review by jr1221 — APPROVED** [review]
Lgtm.

The default thing is nice.

File: scylla-server/migrations/2024-11-10-031516_create_all/down.sql [hunk 2]
```diff
@@ -1,4 +1,8 @@
--- This file should undo anything in `up.sql`
-DROP TABLE "run";
+-- Drop Foreign Keys
```
Comment (RChandler234) [inline scylla-server/migrations/2024-11-10-031516_create_all/down.sql:1]: why is this file changed?

**Review by RChandler234 — CHANGES_REQUESTED** [review]

File: scylla-server/migrations/2024-11-10-031516_create_all/down.sql (hunk 2 above)
Comment (bracyw) [reply scylla-server/migrations/2024-11-10-031516_create_all/down.sql:1]: It didn't work at all, and it needed constraints dropped to work
